packages/navbar/src/react/__stories__/index.story.js

```diff
@@ -83,7 +83,7 @@ storiesOf('Navbar', module).add('desktop', _ => {
           user={
             <BelowRight
               show={
-                <ActionMenu>
+                <ActionMenu origin={ActionMenu.origins.topLeft}>
```

Comment:
Shouldn't this be topRight?

Reply:
Fixed in a different PR

Reply:
Though technically correct given the flopflop